src/unit/test_util.c

```diff
@@ -268,13 +268,13 @@ static int cache_exist(int fd) {
 }
 #endif
 
-int test_reclaimFilePageCache(int argc, char **argv, int flags) {
+int test_utilReclaimFilePageCache(int argc, char **argv, int flags) {
```

Comment:
I don't know if I buy this. We don't really have a naming convention here. If you want to define a naming convention, can you document it in the readme?

Reply:
Unknowingly we all followed the format but only one file's test cases were missed. As you suggested, updated readme file content (Not added specific block for naming convetion). I hope changed content is enough to follow, Please let me know if we need to have a seperate block for naming conventions.